cluster/test-deploy/gcp-crio/vars-origin.yaml

```diff
@@ -0,0 +1,97 @@
+  ]
+
+# URLs and certs
+
+openshift_master_cluster_public_hostname: api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+openshift_master_cluster_hostname: internal-api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+wildcard_zone: apps.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+openshift_master_public_api_url: "https://api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}/"
+openshift_master_public_console_url: "https://api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}/console"
+
+openshift_template_service_broker_namespaces: [ "openshift" ]
+# Disabled until bootstrapping is fixed
+ansible_service_broker_install: false
+openshift_enable_service_catalog: false
+template_service_broker_install: false
+
+console_port: 443
+internal_console_port: 8443
+openshift_master_api_port: "8443"
+openshift_master_console_port: "8443"
+openshift_node_port_range: 30000-32000
+openshift_node_open_ports: [{"service":"Master health check port","port":"8080/tcp"},{"service":"Router stats port","port":"1936/tcp"},{"service":"Allowed open host ports","port":"9000-10000/tcp"},{"service":"Allowed open host ports","port":"9000-10000/udp"}]
+
+# Authentication and authorization
+
+openshift_master_identity_providers: [{"name": "allow_all", "login": "true", "challenge": "true", "kind": "AllowAllPasswordIdentityProvider"}]
+
+# Post config setting sizes
+
+openshift_hosted_router_replicas: 1
+openshift_hosted_registry_replicas: 1
+openshift_schedulable: True
+
+openshift_hosted_registry_storage_gcs_keyfile: "{{ files_dir }}/gce.json"
+openshift_hosted_registry_storage_kind: object
+openshift_hosted_registry_storage_provider: gcs
+openshift_hosted_registry_storage_gcs_bucket: origin-ci-test-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}
+
+container_runtime_docker_storage_type: overlay2
+
+provision_custom_repositories: [
+  {
+    "name": "dockertested",
+    "baseurl": "https://mirror.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://use-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://euw-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror1.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
+        https://gce-mirror2.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/",
+    "sslverify": "false",
+    "sslclientkey": "ops-mirror.pem",
+    "sslclientcert": "ops-mirror.pem",
+    "failovermethod": "priority",
+    "gpgcheck": 0,
+    "gpgkey": "https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-beta https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-openshifthosted"
+  }
```

Comment:
ok so the cbs.centos.org repo mentioned above has cri-tools now. Can you add the following and check how it goes..


{
"name": "CentOS PaaS SIG - Origin 3.10",
"baseurl": https://cbs.centos.org/repos/paas7-openshift-origin310-candidate/x86_64/os/",
"sslverify": "false",
"gpgcheck": 0
}

Reply:
@mrunalp ^

Reply:
Yeah, trying it out. Thanks!